pkgs/tools/misc/atuin/default.nix

```
@@ -41,6 +52,6 @@ rustPlatform.buildRustPackage rec {
     description = "Replacement for a shell history which records additional commands context with optional encrypted synchronization between machines";
     homepage = "https://github.com/ellie/atuin";
     license = licenses.mit;
-    maintainers = with maintainers; [ onsails SuperSandro2000 sciencentistguy ];
+    maintainers = with maintainers; [ onsails SuperSandro2000 sciencentistguy _0x4A6F ];
```

```suggestion
    maintainers = with maintainers; [ SuperSandro2000 sciencentistguy _0x4A6F ];
```
Lets try to keep maintainers not to big.
onsails did the initial commit and since that no commits, should be fine to drop. scientistguy is contributing upstream, so fine to keep.